### PR TITLE
feat: added input three size

### PR DESCRIPTION
**Pull request by vtrbo**

### CODE REVIEW

**Review by baiwusanyu-c — CHANGES_REQUESTED** [review]
How about adding a test for `isError` and `errorMsg `which will be used in the Form component

File: components/Input/__test__/input.spec.ts [hunk 1]
```diff
@@ -20,6 +20,51 @@ afterEach(() => {
...
+				iconPrefix: 'i-prefix',
+				iconSuffix: 'i-suffix'
+			}
+		});
+		expect(instance).toBeTruthy();
+		expect((host as HTMLElement)!.innerHTML.includes('k-input--lg')).toBeTruthy();
+		expect((host as HTMLElement)!.innerHTML.includes('k-input--icon--lg')).toBeTruthy();
+		expect(host.innerHTML).matchSnapshot();
+	});
+
```
Comment (songjianet) [inline components/Input/__test__/input.spec.ts:67]: Maybe it would be better to combine all size tests into one test. ['sm', 'md', 'lg'].forEach(s => {})

**Review by songjianet — CHANGES_REQUESTED** [review]

**Review by baiwusanyu-c — CHANGES_REQUESTED** [review]
image
https://deploy-preview-246--ikun-ui.netlify.app/components/KInput.html#input-slots

**Comment by baiwusanyu-c** [comment]
image
https://deploy-preview-246--ikun-ui.netlify.app/components/KInput.html#sizes

crowded

**Review by baiwusanyu-c — APPROVED** [review]

**Pull request merged**